### PR TITLE
hide help link for non TiDB distro

### PR DESCRIPTION
**Pull request by baurine**
How to build a distro dashboard:

1. Add the following code in the populate.go to pass the compile:

	```
	var Resource = map[string]string{
		"tidb":    "TiDB",
		"tikv":    "TiKV",
		"tiflash": "TiFlash",
		"pd":      "PD",
	}
	```

2. Build UI

    ```
    UI=1 DISTRO_BUILD_TAG=1 make UI
    ```

3. Build all

    ```
    UI=1 DISTRO_BUILD_TAG=1 make
    ```

### CODE REVIEW

File: ui/dashboardApp/layout/signin/index.tsx [hunk 1]
```diff
@@ -172,7 +173,13 @@ function useSignInSubmit(
     } catch (e) {
       if (!e.handled) {
         const errMsg = t('signin.message.error', { msg: e.message })
-        if (e.errCode === 'error.api.user.insufficient_privileges') {
+        if (
+          distro.tidb !== 'TiDB' ||
```
[the remark this answers is not in the document]
Comment (baurine) [reply ui/dashboardApp/layout/signin/index.tsx:177]: Let me try it

[the remark this answers is not in the document]
Comment (baurine) [reply ui/dashboardApp/layout/signin/index.tsx]: Done!

File: ui/lib/utils/i18n.ts [hunk 2]
```diff
@@ -60,4 +60,8 @@ i18next
     },
   })
 
+export function isDistro() {
+  return process.env.REACT_APP_DISTRO_BUILD_TAG !== undefined
```
Comment (baurine) [inline ui/lib/utils/i18n.ts:64]: Tried use `return process.env.REACT_APP_DISTRO_BUILD_TAG !== 1` to judge, but can't pass the ts compile, it says process.env.xxx values are string | null，although in this case, process.env.REACT_APP_DISTRO_BUILD_TAG is a number.

Comment (shhdgit) [reply ui/lib/utils/i18n.ts:64]: Maybe we can export the variables directly, like `const isDistro = process.env.REACT_APP_DISTRO_BUILD_TAG !== undefined`.

**Comment by shhdgit** [comment]
LGTM.

**Review by shhdgit — APPROVED** [review]

File: ui/lib/utils/i18n.ts (hunk 2 above)
Comment (baurine) [reply ui/lib/utils/i18n.ts:64]: Make sense, let me udpate it.

**Pull request merged**